### DISCUSSION TITLE
a git extension?

### DISCUSSION THREAD

**Discussion by KylomaskGamer** [TurboWarp/extensions#771] [Ideas]
I can sync data with thingys

**Comment by PURPL3-CAT** [comment]
Can you be a little more specific? What data do you want to sync?

**Comment by KylomaskGamer** [comment]
like 2 discord bots syncing currencies

**Reply by KylomaskGamer** [reply]
i didnt mean that i just went "DISCORD BOT"

**Reply by yuri-kiss** [reply]
mhm

**Comment by KylomaskGamer** [comment]
wait
i mean
uhhhh
filesystems

**Reply by yuri-kiss** [reply]
smooth..
